src/components/Planning/PlanningGroupsTable/PlanningGroupsTableRow.tsx

```diff
@@ -47,7 +59,14 @@ const PlanningGroupsTableRow: FC<IPlanningProjectsTableProps> = ({ project }) =>
         <div className="project-header-cell-container">
           {/* LEFT */}
           <div className="left">
-            <IconMenuDots size="xs" />
+            <IconMenuDots
+              size="xs"
+              style={{ cursor: 'pointer' }}
```

Comment:
Please use css styling instead of direct styling

Reply:
resolved